com.github.marktext.marktext.json

```diff
@@ -16,7 +16,6 @@
         "--share=ipc",
         "--share=network",
         "--filesystem=home",
-        "--filesystem=host",
```

Comment:
Is there a special reason why you removed this permission? If we don't claim host access we cannot longer access (network) drives or other paths than the home directory.

Reply:
I think we should allow access to the `host` and remove the redundant entry `home` as in PR #26.

Reply:
I removed this permission because of the redundancy. I chose to remove the access to `host` and leave the access to `home` in order to keep minimal permissions by default. I figured that in the case where a user would need to access files out of his home directory (didn't think about network drives though) he could change this permission (with flatseal for example).

Reply:
I don't really know which option we should use. If we remove `host` permissions, the user might need to change permissions locally. Using `host`, we have a bit less security, but the application can also only access files owned by the current user. I guess the best idea, from a user standpoint, is to keep the permission as it is - other Flatpak projects also prefer `host` over `home`.

Could you please resolve the merge conflicts, then we can merge the PR. In the end it doesn't really matter which option you choose because the project is unmaintained.

Reply:
That's done !